papers/urls.py

```diff
@@ -51,6 +48,13 @@
         views.PublisherPapersView.as_view(), name='publisher-papers'),
     url(r'^journal/(?P<journal>\d+)/$',
         views.JournalPapersView.as_view(), name='journal'),
+    # These AJAX views are a trick to avoid displaying JSON output in browser
+    # when going backwards, see #198.
+    url(r'^ajax/search$', views.PaperSearchView.as_view(), name='ajax-search'),
+    url(r'^ajax/r/(?P<researcher>\d+)/(?P<slug>[\w-]*)$', views.ResearcherView.as_view(),
+        name='ajax-researcher'),
+    url(r'^ajax/my-profile$', views.myProfileView,
+        name='ajax-my-profile'),
```

Comment:
I guess we should not need this URL, since `ajax-my-profile` is not used anywhere else, right? Given that the `my-profile` view just reuses the researcher view, it will have the `ajax-researcher` URL in its context, I think.

Reply:
Indeed, well catched, thanks!